chain/syncer.go

```diff
@@ -151,11 +152,20 @@ func NewSyncer(cst *hamt.CborIpldStore, c consensus.Protocol, s syncerChainReade
 // otherwise resolved over the network.  This method will timeout if blocks
 // are unavailable.  This method is all or nothing, it will error if any of the
 // blocks cannot be resolved.
-func (syncer *Syncer) getBlksMaybeFromNet(ctx context.Context, blkCids []cid.Cid) ([]*types.Block, error) {
+func (syncer *Syncer) getBlksMaybeFromNet(ctx context.Context, blkCids []cid.Cid) ([]types.TipSet, error) {
 	ctx, cancel := context.WithTimeout(ctx, blkWaitTime)
 	defer cancel()
 
-	return syncer.fetcher.GetBlocks(ctx, blkCids)
+	tss, err := syncer.fetcher.FetchTipSets(ctx, types.NewTipSetKey(blkCids...), 1)
```

Comment:
Since we don't have "Initial sync"/"caught up" modes this will just behave like it did before.